lib/chef/resource/windows_pagefile.rb

```diff
@@ -82,14 +91,14 @@ class WindowsPagefile < Chef::Resource
 
         if automatic_managed
           set_automatic_managed unless automatic_managed?
-        else
+        elsif automatic_managed == false
           unset_automatic_managed if automatic_managed?
+        else
```

Comment:
I'm assuming there is a reason for this, but I don't have context here. Why did we add an `else` here? It seems like if you dont want `automatic_managed` then you would include all the code you had before.

Reply:
What I am trying to accomplish is ensure that under the :set action if you aren't setting a pagefile to un/managed then you're either creating a new pagefile if it doesn't exist or adjusting its settings.

Reply:
but if you are setting it to `unmanaged` its assumed it should skip the creation if it does not exist?

Reply:
Correct. And if you try to create or set a pagefile without setting unmanaged, the drive gets set to unmanaged for you as a function of those actions.

Reply:
what if you try to create or set a pagefile and set it to unmanaged?

Reply:
if you create a pagefile at c:\pagefile.sys and then set the drive to unmanaged you get a System Managed drive where there is no pagefile on that drive but you are going to defer to the OS to manage a pagefile as need be. In the case where you have only a C drive then auto-managed and system-managed are the same thing.